src/Input/Input.js

```diff
@@ -34,47 +17,78 @@ const Input = styled.input`
   transition-delay: 0s;
   font-size: inherit;
   margin-bottom: 10px;
-
-  ${props => props.fullWidth && css`
-    width: 100%;
-  `};
-
-  &:focus {
-    border-bottom: 1px solid #0094d2;
+  
+  ${props => props.focus && css`
```

Comment:
Focus should be a CSS selector. The animation does not work when you leave the input

Reply:
Solved!

The order in css matters xD! Sorting the rules applied correctly, works like a charm!